src/components/Element/index.ts

```diff
@@ -299,7 +295,7 @@ class Element extends Events {
 
     protected _destroyed: boolean;
 
-    protected _parent: any;
+    protected _parent: Element; // eslint-disable-line no-use-before-define
```

Comment:
Why does eslint need to be disabled for this line?

Reply:
Because otherwise ESLint that `no-use-before-define` error is triggered. Because `_parent` is of type Element but part of the `Element` definition itself. There might be a better way around it, but for now, I disabled the error.